tools/build/Makefile.vars

```diff
@@ -335,8 +335,8 @@ $(call add-template,$(FLOW_METATYPE_BUILTINS_H_IN),$(FLOW_METATYPE_BUILTINS_H))
 NODE_TYPE_SCHEMA := $(top_srcdir)data/schemas/node-type-genspec.schema
 NODE_TYPE_SCHEMA_DEST := $(build_flowdatadir)schemas/node-type-genspec.schema
 
-BOARD_DETECT := $(top_srcdir)data/jsons/board_detect.json
-BOARD_DETECT_DEST := $(build_datadir)board_detect.json
+BOARD_DETECT := $(top_srcdir)data/jsons/99_board_detect.json
+BOARD_DETECT_DEST := $(build_datadir)/boards/99_board_detect.json
```

Comment:
Roger that

On Fri, Nov 13, 2015, 20:14 Gustavo Sverzut Barbieri <
notifications@github.com> wrote:

> In tools/build/Makefile.vars
> <https://github.com/solettaproject/soletta/pull/1055#discussion_r44840628>
> :
>
> > @@ -335,8 +335,8 @@ $(call add-template,$(FLOW_METATYPE_BUILTINS_H_IN),$(FLOW_METATYPE_BUILTINS_H))
> >  NODE_TYPE_SCHEMA := $(top_srcdir)data/schemas/node-type-genspec.schema
> >  NODE_TYPE_SCHEMA_DEST := $(build_flowdatadir)schemas/node-type-genspec.schema
> >
> > -BOARD_DETECT := $(top_srcdir)data/jsons/board_detect.json
> > -BOARD_DETECT_DEST := $(build_datadir)board_detect.json
> > +BOARD_DETECT := $(top_srcdir)data/jsons/99_board_detect.json
> > +BOARD_DETECT_DEST := $(build_datadir)/boards/99_board_detect.json
>
> last one is always bad, usually people call this "50-default.json", so we
> know it is the default we provide and sits right in the "middle" of the
> range. People can add higher or lower priority versions as they see fit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/solettaproject/soletta/pull/1055/files#r44840628>.
>